html/webappapis/scripting/events/event-handler-processing-algorithm.html

```diff
@@ -5,21 +5,39 @@
  <body>
  <div id="foo" style="width: 100px; height: 100px; background-color: black"></div>
  <script>
+
+ // Historically mouseover was special in the spec, but now it is not. See https://github.com/whatwg/html/pull/2398.
  async_test(function(t) {
      var ev = new Event('mouseover', {cancelable: true});
-     document.getElementById("foo").onmouseover = t.step_func(function() { return true });
+     document.getElementById("foo").onmouseover = t.step_func(function() { return false });
+     document.getElementById("foo").dispatchEvent(ev);
+     assert_equals(ev.defaultPrevented, true)
+     t.done();
```

Comment:
This was already an async_test, but would like to note my preference for plain tests where possible. Otherwise I end up looking for the async bit. No change needed.